www/js/main.js

```diff
@@ -448,7 +455,7 @@ function setBound() {
 
 function vibrate(duration) {
     if (typeof duration === 'undefined') duration = 500;
-    if (typeof navigator.notification !== 'undefined') navigator.notification.vibrate(duration);
+    if (typeof navigator.notification !== 'undefined') navigator.vibrate(duration);
```

Comment:
@daydreamheart can we just check `navigator` instead of `navigator.notification` since we not using the `notification` anyway?

Reply:
Yes, I changed it to `navigator`, it works as well that way.